### DISCUSSION TITLE
请问用mpv-lazy 正常播放hdr视频的姿势

### DISCUSSION THREAD

**Discussion by jiang-yuan** [hooke007/mpv_PlayKit#164] [Q&A 问答] [answered]
刚入了sanc h40pro 34寸的 hdr400的屏幕,虽然没600效果一般,但是据说开了看视频也比sdr好一点
我直接在win11显示设置里面开那个hdr开关连普通显示效果都不好,直接泛白我立马关这个全局的设置了
有啥方式是只在mpv里面播放hdr视频的时候让显示器开启hdr模式的吗?我显示器已经设置为自动hdr了

**Comment by hooke007** [comment, marked as answer]
A3
https://hooke007.github.io/mpv-lazy/mpv_lazy_d04#q-mpv%E4%B8%8D%E8%BD%ACsdr%E8%A7%82%E7%9C%8Bhdr

_标准的hdr400没有任何hdr的效果，强开hdr属于自欺欺人的脑放行为，它的标准色域就是sdr，和hdr600是从0到1的区别（sdr也属于0）_

**Reply by jiang-yuan** [reply]
那算了,我显示器峰值亮度就这么高,懒得改配置了
